### ISSUE TITLE
Show file size

### ISSUE THREAD

**Issue by maxoku** [MicroDroid/youtube-download-button#10]
It would be nice if downloads showed size of files that are listed. Size of a file is a very useful information unlike other things that are shown (e.g. @ 0.5Mbps ~25fps).

**Comment by MicroDroid** [comment]
Agreed, bitrate along with time can be used to estimate file size. Do you mind PRing that?
